.github/workflows/ci.yml

```
@@ -0,0 +1,53 @@
+name: Go
+
+on: [push, pull_request]
+
+jobs:
+  shellcheck:
```

wdyt of renaming the entire job to this, which is more inclusive with other tasks than shellcheck?
```suggestion
  static-analysis:
```